.travis.yml

```diff
@@ -10,7 +10,7 @@ before_install:
   - sleep 3
 install:
   - sudo apt-get install python-tk python3-tk
-  - python -m pip install -U setuptools pip
+  - python -m pip install -U setuptools pip importlib-metadata
```

Comment:
Why do you need another dependancy ?

Reply:
The Travis build on Python 3.7 seems to fail without that. Same issue with `ttkthemes` build.